src/DIRAC/FrameworkSystem/Service/ComponentMonitoringHandler.py

```diff
@@ -352,6 +352,14 @@ def export_getLog(self, host):
 
     return S_OK(result['Value'])
 
+  types_getLogs = []
+
+  def export_getLogs(self):
```

Comment:
This should probably be restricted to some admin roles by default

Reply:
In the `ConfigTemplate.cfg` for the `ComponentMonitoring` `Default` authorization is set to `ServiceAdministrator`, I don't think the data obtained by this method needs to be further protected in the code using `auth_getLogs`.

Reply:
Fair enough ! :-)